modules/opensearch/css/opensearch_autocomplete_admin_form.css

```diff
@@ -0,0 +1,56 @@
+#edit-opensearch-autocomplete .form-type-radio {
```

Comment:
This CSS is somewhat broken in my opinion. I think it would be OK to drop it.

Reply:
My fault. I fixed it.

![screen shot 2017-11-16 at 10 24 19](https://user-images.githubusercontent.com/514013/32883708-5dd9634e-cab8-11e7-9352-04804a58cf3f.png)

I think we should keep it.